egs/how2/mt1/run.sh

```diff
@@ -29,8 +30,12 @@ n_average=5                  # the number of NMT models to be averaged
 use_valbest_average=true     # if true, the validation `n_average`-best NMT models will be averaged.
                              # if false, the last `n_average` NMT models will be averaged.
 
+# cascaded-ST related
```

Comment:
Do we need this for MT?
(If you intend to leave these lines to make run.sh similar to ST, that is fine)

Reply:
This is the minimal design for cascaded ST.

Reply:
OK.